src/routes/auth/create-new-user.e2e.test.js

```diff
@@ -0,0 +1,191 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const { fixtures } = require('../../../test/utils')
+
+describe('POST /auth/create-user', () => {
+	const validPassword = 'validPaswword123$'
+	describe('Failure', () => {
+		let signupInfo
+		let inviteToken
+
+		before(async () => {
+			signupInfo = {
+				firstName: faker.name.firstName(),
+				lastName: faker.name.lastName(),
+				email: faker.internet.email(),
+				password: faker.internet.password()
+			}
+			inviteToken = fixtures.generateInviteToken(signupInfo.email)
+			await fixtures.insertUserInvite(
+				{email: signupInfo.email})
+		})
+
+		it('should throw an error if email is invalid', async () => {
```

Comment:
so our service test will deal with the throwing of errors, here is mostly about returning status codes and the expected error response, not throwing.  

also kindly have inner describes `Failure` and `Success`

and lets test the lack of the required fields as well

Reply:
@jocrah Please I'm not clear on the inner describes. I have inner describes in this PR, should every test have it's own describe?

Reply:
i see a Success describe block but not a Failure block hence the review

Reply:
You must have missed it @jocrah https://github.com/tbf-challenge/teamwork-be/blob/198-test-create-user-endpoint/src/routes/auth/create-new-user.e2e.test.js#L7

Reply:
haha true, i must have. these eyes of mine. sincere apologies

Reply:
haha, it happens